### PR TITLE
Adjust copied to clipboard position in NewAddress

### PR DESCRIPTION
**Pull request by erkarl**
This only changes the styling of the initial `NewAddress` view. Closes: https://github.com/lightninglabs/lightning-app/issues/682

Before:
before

After:
after

### CODE REVIEW

**Comment by erkarl** [comment]
Closing as this needs some additional thought.

**Comment by valentinewallace** [comment]
@erkarl I talked to our designer and we think this solution is fine for now :) Reopening.

File: src/view/new-address.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const styles = StyleSheet.create({
     marginBottom: 40,
   },
   copied: {
-    bottom: 180,
+    bottom: 60,
```
Comment (valentinewallace) [inline src/view/new-address.js]: V simple.

**Comment by erkarl** [comment]
@valentinewallace CLA updated.

File: src/view/new-address.js (hunk 1 above)
Comment (tanx) [reply src/view/new-address.js]: Setting `color={color.notifyDark}` on `CopiedNotification` in line 60 is probably cleaner: https://github.com/lightninglabs/lightning-app/pull/688/files#diff-7ed4897f600feac8ff2579bda31e993aR60

**Comment by tanx** [comment]
Closing in favor of #707

**Pull request closed without merging**